src/App.tsx

```diff
@@ -1,16 +1,45 @@
 import React from 'react';
 import './App.scss';
 
-class App extends React.Component {
-  addOne = () => {};
+type State = {
+  count: number,
+};
 
-  add100 = () => {};
+class App extends React.Component<{}, State> {
+  state = {
+    count: 0,
+  };
 
-  increase = () => {};
+  addOne = () => {
+    this.setState((state) => ({
+      count: state.count + 1,
+    }));
+  };
+
+  add100 = () => {
+    this.setState((state) => ({
+      count: state.count + 100,
+    }));
+  };
+
+  increase = () => {
+    this.addOne();
+
+    if (!(this.state.count % 5)) {
```

Comment:
in such cases better to write it explicitly 

```suggestion
    if (this.state.count % 5 === 0) {
```